test/Incremental/multi-file/Inputs/Base.swift

```diff
@@ -0,0 +1,39 @@
+open class OpenBase {} // expected-provides {{OpenBase}}
```

Comment:
Wild idea: Can we take existing projects, such as ones in the source compatibility repo, automatically generate these expectations, and then save that for stability tests that would automatically tell us if something changed?